paddlets/datasets/tsdataset.py

```diff
@@ -778,10 +778,10 @@ def load_from_csv(
         known_cov_cols: Optional[Union[List[str], str]] = None,
         static_cov_cols: Optional[Union[List[str], str]] = None,
         freq: Optional[Union[str, int]] = None,
-        fill_missing_dates: bool = False,
-        fillna_method: str = "pre",
-        fillna_window_size: int = 10,
-        drop_tail_nan: bool = False,
+        fill_missing_dates: Optional[bool] = False,
+        fillna_method: Optional[str] = "pre",
+        fillna_window_size: Optional[int] = 10,
+        drop_tail_nan: Optional[bool] = False,
```

Comment:
Line781 - Line784 should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional